src/js/text/twoSided.dot.html

```diff
@@ -0,0 +1,40 @@
+<div class="brexit brexit--nav-controls">
+    <div class="brexit__slider-container js-slider-container">
+        <div class="brexit__content brexit__slider">
+            <h1 class="brexit__header">
+                {{=it.data.answer_1_title}}
+            </h1>
+            <p>
+                {{=it.data.answer_1_body}}
+            </p>
+        </div>
+        <div class="brexit__content brexit__slider">
+            <h1 class="brexit__header">
+                {{=it.data.question}}
+            </h1>
+        </div>
+        <div class="brexit__content brexit__slider">
+            <h1 class="brexit__header">
+                {{=it.data.answer_2_title}}
+            </h1>
+            <p>
+                {{=it.data.answer_2_body}}
+            </p>
+        </div>
+    </div>
+    <div class="brexit__content brexit__feedback">
+        {{#def.feedback}}
+    </div>
+    <a data-answer="one" class="brexit__nav-control brexit__nav-control--text brexit__nav-control--left js-go-to-answer">
+        <div class="brexit__nav-control__label">{{=it.data.answer_1_title}}</div>
+    </a>
+    <a data-answer="two" class="brexit__nav-control brexit__nav-control--text brexit__nav-control--right js-go-to-answer">
+        <div class="brexit__nav-control__label">{{=it.data.answer_2_title}}</div>
+    </a>
+    <a data-answer="two" class="brexit__nav-control brexit__nav-control--left js-back-to-question u-hidden">
```

Comment:
Replace classes `brexit__nav-control--left` and `brexit__nav-control--right` with `brexit__nav-control--prev` and `brexit__nav-control--next`? Otherwise chevron SVGs don't appear in the buttons